bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/window.xml

```diff
@@ -16,7 +16,7 @@
 		<description>@text/thing-type.velux.window.description</description>
 		<category>Window</category>
 		<channels>
-			<channel id="position" typeId="position"></channel>
+			<channel id="position" typeId="windowPosition"></channel>
```

Comment:
This becomes a breaking change for existing things created with UI.

Reply:
I don't want to make a breaking change! The only reason why I created a new channel type is because it should have a different label and description text. So I will revert adding this new channel type and instead just override the texts for the channel.

Reply:
I tend to agree with you. That being said, I let you evaluate the balance between a better "matching" and a breaking change.
Of course, we can accept a breaking change, especially if it is documented in the release notes.

Reply:
Ok. I think we should change it. The previous category setting was wrong, but in earlier versions of OH was relatively unimportant. However as you know, the OH v3.x UI places much more emphasis on the correct semantic categorisation of things and channels within a house, so I think we should correct this error.

PS the limitMinimum / limitMaximum channels were also in category 'blinds', so to be consistent I have removed the category, since those channels are equally applicable to 'windows' and 'blinds' things.